vc/pokecrystal11_vc.patch.template

```diff
@@ -0,0 +1,698 @@
+Mode = 2
+Address = {HEX @ 4}
+Type = 12
+[to_play2_battle]
+Mode = 2
+Address = {HEX @ 4}
+Type = 13
+[ret_heya]
+Mode = 2
+Address = {HEX @ 4}
+Type = 14
+
+
+
+
+
+;ROM:3FBCD                 ld      b, $3E ; '>'
+;ROM:3FBCF                 inc     de
+;ROM:3FBD0                 call    unk_2D55
+;ROM:3FBD3                 xor     a
+;ROM:3FBD4                 ld      [byte_FFD2], a
+;ROM:3FBD6                 ld      [byte_FF4A], a
+;ROM:3FBD8                 call    unk_31C2
+;ROM:3FBDB                 call    unk_2FE2
+;
+;ROM:3FBCD:  06 3E 13 CD
+
+;0003fbb5h: 06 3E 13 CD                                     ; .>.?
+
+[fight begin]                        
+Mode = 11                       
+Type = 0                        
+Index = 1                       
+Address = {HEx @}            
+Fixcode={db SCREEN_HEIGHT_PX}        
+              
+;12 1b 0b 79 b0  find next C9
+[BiographySave_ret]
+Mode = 2
+Address = {HEX @ 4}
+Type = 60
+
+
+; print forbid 1
+;ROM:1758D                 ld      a, [byte_FFA9]
+;ROM:1758F                 and     2
+;ROM:17591                 jr      nz, unk_75B4
+;ROM:17593                 ld      a, [byte_FFA9]
+;ROM:17595                 and     1                      ;e6 01
+;ROM:17597                 jr      nz, unk_75A1
+;    
+; change "and 1" to "and 0"
+;00017595h: E6 01 20 08 CD BF 75 CD 2E 03 18 E9 FA 57 CE F5 ; ? .涂u?..辁W熙
+;00016c76h: E6 01 20 08 CD A0 6C CD 5A 04 18 E9 FA 63 CF F5 ; ? .蜖l蚙..辁c硝
+[print forbid 1]                                                   
+Mode = 1                                                                                                                  
+Address = {hex @ 4}
+Fixcode={patch}      
+
+
+                            
+[print forbid 2]                                                                           
+Mode = 6                                                                                       
+Type = 0                                                                                       
+Address = {hex @ 4}                                                                        
+MemAddress={hex hJoyPressed}                                                                           
+Fixcode={db NO_INPUT}                                                                                  
+ConditionType = 0                                                                              
+ConditionValueA = {dws_ wWindowStackPointer wWindowStackPointer+1 wMenuJoypad wMenuSelection wMenuSelection wMenuCursorY hJoyPressed hJoyPressed hJoyPressed hJoyPressed}                   
+ConditionValueB = {dws_ ==                  ==                    ==          >=             <=             ==           !=          !=          !=          !=         }                   
+ConditionValueC = {dws_ 0xdd                0xd3                  A_BUTTON    0x00           0x0f           0x03         D_DOWN      D_UP        B_BUTTON    NO_INPUT   }                   
```

Comment:
This is a very wack expression. Do we know what menu it corresponds to (the other four are unown dex, cianwood photo studio, mail and the pokedex), and why the window stack pointer is set as it is? This sounds like something that a slight menu restructure would break.

Not an easy fix and probably not going to be handled in this PR but the more we learn about this patch the wackier it gets.

Reply:
Possibly for printing PC boxes?